frontend/public/components/utils/index.tsx

```diff
@@ -78,3 +80,19 @@ export const enum EnvType {
   ENV = 0,
   ENV_FROM = 1
 }
+
+export type FirehoseResource = {
+  kind: K8sResourceKindReference;
+  name?: string;
+  namespace?: string;
+  isList?: boolean;
+  selector?: Selector;
+  prop: string;
+  namespaced?: boolean,
+};
+
+export type FirehoseResult = {
```

Comment:
We should update `overview/index.tsx` to use this:

https://github.com/openshift/console/blob/master/frontend/public/components/overview/index.tsx#L1065-L1068

Reply:
`overview/index.tsx` is now updated. I've changed `FirehoseResult` type definition a bit to fit `overview/index.tsx` better